src/shutdown/shutdown.c

```diff
@@ -305,6 +342,16 @@ int main(int argc, char *argv[]) {
         (void) cg_get_root_path(&cgroup);
         in_container = detect_container() > 0;
 
+        /* If the logging messages are going to KMSG, and if we are not running from a container,
+         * then try to update the sysctl kernel.printk current value in order to see "notice" messages;
+         * This current log level is not updated if already big enough.
+         */
+        if (!in_container && !IN_SET(log_get_target(), LOG_TARGET_AUTO,
```

Comment:
test is reversed or am i blind? i.e. needs to be `IN_SET()`, and not `!N_SET()`, no?

Reply:
Ouch!!!!